README.textile

```diff
@@ -1,19 +1,21 @@
+"!https://badge.waffle.io/hbz/lobid.png?label=ready&title=Working!":https://waffle.io/hbz/lobid
+
 h1. Lobid
 
 Linking Open Bibliographic Data.
 
-This software implements "http://lobid.org":http://lobid.org.
+This is only the landing page of lobid org.
+For the repos with the software implementations see:
+
+"https://github.com/hbz/lobid-resources":lobid-resources implements "https://lobid.org/resources":https://lobid.org/resources.
+"https://github.com/hbz/lobid-organisations":lobid-organisations implements  "https://lobid.org/organisations":http://lobid.org/organisations.
+"https://github.com/hbz/lobid-gnd":lobid-gnd implements "https://lobid.org/gnd":https://lobid.org/gnd.
+"http://github.com/hbz/lobid-nwbib":lobid-nwbib implements "https://lobid.org/nwbib":https://lobid.org/nwbib.
```

Comment:
It's https://github.com/hbz/nwbib/ and https://nwbib.de (but I don't think we even need to mention it here)

Reply:
Otherwise +1